mypy/checker.py

```diff
@@ -1916,10 +2045,12 @@ def visit_yield_expr(self, e: YieldExpr) -> Type:
     def check_subtype(self, subtype: Type, supertype: Type, context: Context,
                       msg: str = messages.INCOMPATIBLE_TYPES,
                       subtype_label: str = None,
-                      supertype_label: str = None) -> None:
+                      supertype_label: str = None) -> bool:
```

Comment:
I think the two things that made me feel weird about this usage are:
- Most of the `check_*` methods return `None` (but it looks like there's also a decent percentage that return all sorts of other things).
- The return value just comes directly from an `is_subtype` call, so it feels like it makes a bit more sense to refactor out the error formatting code so it can be called separately.

If you don't feel the same way I'm fine with leaving this as-is.